packages/cli/test/commands/migrate/validate.test.ts

```diff
@@ -53,51 +53,35 @@ describe('Task: validate', () => {
   });
 
   test('pass with package.json', async () => {
-    const options = createMigrateOptions(project.baseDir);
-    const tasks = [validateTask(options, logger)];
-
-    await listrTaskRunner(tasks);
+    await runValidate(basePath, logger);
     expect(cleanOutput(output, project.baseDir)).toMatchSnapshot();
   });
 
   test('error if no package.json', async () => {
-    const options = createMigrateOptions(project.baseDir);
-    const tasks = [validateTask(options, logger)];
-
-    rmSync(resolve(project.baseDir, 'package.json'));
-
-    await expect(() => listrTaskRunner(tasks)).rejects.toThrowError(`package.json does not exists`);
+    rmSync(resolve(basePath, 'package.json'));
```

Comment:
fixurify has an API for tmp file creation you should leverage

Reply:
delete project.files['package.json']

Reply:
Tested on my local. delete project.files['package.json'] won't work, since `package.json` is automatically generated by project.write(). `package.json` is not listed in the project.files array.

Reply:
can you just delete AFTER project.write()